app/controllers/api/v4/runs/users_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Api::V4::Runs::UsersController < Api::V4::ApplicationController
+  before_action :set_run
+  before_action only: [:destroy] do
+    doorkeeper_authorize! :delete_run
+  end
+
+  def destroy
+    if @run.update(user: nil)
+      head 205
```

Comment:
Nice call on the 205 (vs 204), can you update `Api::V4::RunsController#destroy` as well?